### ISSUE TITLE
Add Grapple Conditon and Marker

### ISSUE THREAD

**Issue by sIKE23** [sIKE23/Mage-Wars#355]
@ACG8 

When one creature Grapples another, give that creature a Grapple marker and attach them to each other.

Thoughts?

**Comment by ACG8** [comment]
Well, we definitely can't just attach them to each other. That is, we can't create a Grapple relation that is symmetric between the two creatures; one will have to be the grappler and one will have to be the grapplee, otherwise it is impossible to determine how to orient them with respect to each other.

**Comment by sIKE23** [comment]
One will have a Grapple Marker....

**Comment by ACG8** [comment]
Right, but in order to orient the cards we will need to determine which one is moved in response to the other. Basically, I am thinking handle it like attach/bind/mount, but with grapple.

That said, we are running out of directions to splay the cards.

**Comment by sIKE23** [comment]
Well lets push them together like Mount, but use card.filter to show that
they are grappled....

//FC
